src/components/mote/components/innkalling/DialogmoteMoteStatusPanel.tsx

```diff
@@ -40,52 +43,68 @@ const getIcon = (vilMote?: boolean): string => {
   }
 };
 
-const ParticipantInfo = (): ReactElement => {
-  const detailTextArbeidstaker = "Jeg er til behandling den dagen";
-  const detailTextArbeidsgiver = "Pina d grett";
+interface ParticipantProps {
+  dialogmote: DialogmoteDTO;
+  bruker: Brukerinfo | undefined;
```

Comment:
kan være `bruker?: Brukerinfo`

Reply:
Hvis flere enn meg er usikre på dette med `?`, `null`, og `undefined`:
https://stackoverflow.com/questions/57243477/can-an-optional-parameter-be-null-in-typescript

tld;dr:
`?` -> Kan utelates, men kan kun ha verdi eller være `undefined`, ikke `null`
`type | undefined` -> Kan ikke utelates, og kan kun ha verdi eller være `undefined`, ikke `null`
`type | null` -> Kan ikke utelates, og kan kun ha verdi eller være `null`, ikke `undefined`
`type | undefined | null` -> Kan ikke utelates, kan ha verdi, være `undefined`, eller `null`.
`? type | undefined | null` -> Kan utelates, kan ha verdi, være `undefined`, eller `null`.

Reply:
Fin oppklaring 👌 Har ikke vært så obs på forskjellen mellom `?` og `type | undefined` før. Da tenker jeg `bruker: Brukerinfo | undefined;` er mest riktig her siden man bør sende inn en bruker-prop til `ParticipantInfo`.

Reply:
Liten digresjon, men da jeg googlet noe rundt dette tidligere da jeg fikk PR-kommentar:
`Use undefined. Do not use null.`  fra typescript guidelines:
https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined

Reply:
Bra oppsummering JM. Tenker også at det blir mest riktig slik som det her, ettersom brukerinfo ikke skal være optional. 

En muligens bedre fiks er å initiere en "tom" initialstate i redux, slik at man aldri kan få undefined.

Fra

```
export interface NavbrukerState {
  data?: Brukerinfo;
}
(Ingen initial state)
```

Til
```
export interface NavbrukerState {
  data: Brukerinfo;
}

const initialState = {
  data: {
    navn: '',
    kontaktinfo: {
      fnr: '',
      skalHaVarsel: false
    },
    arbeidssituasjon: ''
  }
}

const navbruker: Reducer<NavbrukerState> = (state = initialState, action) => {
```
Eller noe i den dur.